### PR TITLE
Dimond sponsors page. See #322

### PR DESCRIPTION
**Pull request by GheorghitaVlad**

### CODE REVIEW

File: _html/scss/modules/_dimonds-sponsors-content.scss [hunk 1]
```diff
@@ -0,0 +1,39 @@
+.dimonds-background {
+    background: url('../images/backgrounds/dimond-sponsors-bg.png') no-repeat;
```
Comment (AlexandruSateanuHoria) [inline _html/scss/modules/_dimonds-sponsors-content.scss]: use backgrounds variable ($backgrounds-path)

Comment (bajenaru) [inline _html/scss/modules/_dimonds-sponsors-content.scss]: You already have this image in background as footer-background.png. Please use that so we don't have duplicates.

File: _html/scss/modules/_dimonds-sponsors-content.scss [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+    background-size: cover;
+
```
Comment (bajenaru) [inline _html/scss/modules/_dimonds-sponsors-content.scss]: Remove empty row.

File: _html/templates/dimond-sponsors-page.php [hunk 3]
```diff
@@ -0,0 +1,8 @@
+<?php
+
+include "inc/header.inc.php";
+
+include "modules/dimond-sponsor-content.php";
```
Comment (bajenaru) [inline _html/templates/dimond-sponsors-page.php]: You have a typo here. Please rename everything that has dimond into diamond. Both in SCSS and PHP files. Thanks :).

**Pull request merged**